src/overlays/actors/ovl_Bg_Spot09_Obj/z_bg_spot09_obj.h

```diff
@@ -6,6 +6,8 @@
 
 struct BgSpot09Obj;
 
+typedef void (*BgSpot09ObjActionFunc)(struct BgSpot09Obj*, GlobalContext*);
```

Comment:
Is this used?

Reply:
oh yeah, i made it cause I was gonna typedef these weird functions that run in a loop, but realized it doesnt really work. forgot to remove it